Dataform/.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+**PLEASE READ THIS ENTIRE MESSAGE**
+
+Hello, and thank you for your contribution! Please note that this repository is
+a read-only split of `googleapis/google-cloud-php`. As such, we are
+unable to accept pull requests to this repository.
+
+We welcome your pull request and would be happy to consider it for inclusion in
+our library if you follow these steps:
+
+* Clone the parent client library repository:
+
+```sh
+$ git clone git@github.com:googleapis/google-cloud-php.git
+```
+
+* Move your changes into the correct location in that library. Library code
+belongs in `Dataform/src`, and tests in `Dataform/tests`.
+
+* Push the changes in a new branch to a fork, and open a new pull request
+[here](https://github.com/googleapis/google-cloud-php).
```

Comment:
nit, but it would be clearer if we sent them to the pull request landing page:
```suggestion
[here](https://github.com/googleapis/google-cloud-php/pulls).
```

Reply:
When you push up a branch, the main landing page for the repo will show a widget near the top with "your recently pushed branches." It gives you a button to compare and open a pull request. Given that, I think this is currently fine.